storm/src/main/storm/mesos/NimbusScheduler.java

```diff
@@ -102,7 +102,6 @@ public void statusUpdate(SchedulerDriver driver, TaskStatus status) {
       case TASK_LOST:
       case TASK_ERROR:
         LOG.info("Received status update: {}", taskStatusToString(status));
```

Comment:
Looks good mostly -- but with 3 identical lines I'm think we can construct the message before the switch and then just log it.  i.e., `String msg = String.format(...);`